crates/ruff_linter/src/rules/flake8_pyi/rules/bad_generator_return_type.rs

```diff
@@ -76,12 +98,6 @@ pub(crate) fn bad_generator_return_type(
 
     let name = function_def.name.as_str();
 
-    let better_return_type = match name {
-        "__iter__" => "Iterator",
-        "__aiter__" => "AsyncIterator",
-        _ => return,
-    };
```

Comment:
@AlexWaygood - I removed this since it's enforced in `let (method, module, member) = {` below.